components/motor/fake/motor.go

```diff
@@ -21,7 +21,10 @@ import (
 	rdkutils "go.viam.com/rdk/utils"
 )
 
-const modelName = "fake"
+const (
+	modelName     = "fake"
+	defaultMaxRpm = 1000
```

Comment:
The max_rpm in fake.json used to be 100, and the new value is 10 times larger. Is that change on purpose? I doubt it affects anything important, but it does seem a little unusual.

Reply:
updated to a 100